.gitignore

```diff
@@ -19,6 +19,7 @@ vendor
 !/mod/dashboard/
 !/mod/developers/
 !/mod/diagnostics/
+!/mod/discussion/
```

Comment:
Can we call this "discussions" plural? Would like to standardize on plural form for content plugins.